src/daemon/daemon.cpp

```diff
@@ -1256,6 +1358,32 @@ catch (const std::exception& e)
     status_promise->set_value(grpc::Status(grpc::StatusCode::FAILED_PRECONDITION, e.what(), ""));
 }
 
+void mp::Daemon::list_networks(const ListNetworksRequest* request, grpc::ServerWriter<ListNetworksReply>* server,
+                               std::promise<grpc::Status>* status_promise) // clang-format off
+try // clang-format on
+{
+    mpl::ClientLogger<ListNetworksReply> logger{mpl::level_from(request->verbosity_level()), *config->logger, server};
+    ListNetworksReply response;
+    config->update_prompt->populate_if_time_to_show(response.mutable_update_info());
+
+    auto iface_list = config->factory->list_networks();
+
+    for (auto iface : iface_list)
```

Comment:
No copy needed, `const auto&` would be enough.

Reply:
Done.